src/components/featured-arcgis-layer-manager/featured-arcgis-layer-manager-component.jsx

```diff
@@ -0,0 +1,29 @@
+import { LAND_HUMAN_PRESSURES_IMAGE_LAYER } from 'constants/layers-slugs';
+import { setLayerOrder, setOpacity } from 'utils/arcgis-layer-manager-utils';
+
+const ArcgisLayerManager = ({ map, activeLayers, isLandscapeMode }) => {
```

Comment:
I think this is a good place to extend the functionality of the `ArcgisLayerManager` component by creating a param where we can inject some function call inside the layers loop.
That way instead of duplicating the component forcing us to maintain both of them (and eventually more if we end up creating more scenes) we would be extending the functionality as needed.
I can imagine something like:

on `featured-arcgis-layer-manager-component.jsx`:

```jsx
import LayerManager from `arcgis-layer-manager`;
const FeaturedGlobeLayerManager = ({map, activeLayers, customFunctions}) => {
  return <LayerManager map={map} activeLayers={activeLayers} customFunctions={customFunctions} />
}
```
on `featured-arcgis-layer-manager.js`:

```js
const hideHumanPressuresOnLandscape = (layer, setActive) => {
if(layer.title === LAND_HUMAN_PRESSURES_IMAGE_LAYER) {
    layer.visible = isLandscapeMode && setActive;
 }
}
const customFuntions = [
  hideHumanPressuresOnLandscape
]
```

on `arcgis-layer-manager-component`:

```js
const ArcgisLayerManager = ({ map, activeLayers, customFunctions }) => {
  layers.items.forEach(mapLayer => {
  // ...layer manager stuff
  customfunctions.forEach(fn => fn(mapLayer))
  })
}
```
With something along those lines we would have a place to inject custom functionality or conditionals dependent on the scene needs but would have only one component with this layer toggling functionality that is paramount for the app working and is kinda complex and ugly (and that's another story that we might want to address in the future :)

Reply:
That looks much better, changes are in the repo now. Also, I removed `featured-arcgis-layer-manager` and `data-arcgis-layer-manager` components, and I moved `customFunctions` to the `featured-globe.js` container because having these two separate components seems to be redundant now. Thank you for this hint, I like it better! :1st_place_medal:  :sunglasses: